Sources/App/Models/Bus.swift

```diff
@@ -43,6 +43,7 @@ final class Bus: Hashable, Model {
 		///   - id: An identifier that’s used to update location data dynamically.
 		///   - date: A timestamp that indicates when the location datum was originally collected.
 		///   - coordinate: The geospatial coordinate that’s associated with the location datum.
+		///		- route: The route that the bus is taking associated with the location datum.
```

Comment:
This documentation comment should only mention parameters in the relevant initializer. You didn’t add `route` as a new parameter, so you don’t need to change the documentation comment.

Reply:
This is an artifact from when I thought Bus and Buses were related. This has been removed.